### ISSUE TITLE
Timeline: Does not work with PF CSP enabled

### ISSUE THREAD

**Issue by sasynkamil** [primefaces/primefaces#5871]
## 1) Environment
- PrimeFaces version: 8.0.1
- Does it work on the newest released PrimeFaces version? Version 8.0 is correct.
- Does it work on the newest sources in GitHub? Not, also 9.0-SNAPSHOT broken
- Application server + version: used jetty in primefaces-test project
- Affected browsers: all

## 2) Expected behavior
No error in browser sosole, Timeline should work with enabled PF CSP
...

## 3) Actual behavior
Timeline not displayed, error in browser console:
Uncaught EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self' 'nonce-MTlmZDUyODMtN2RiNi00OWQ1LTgzY2UtZjIzZDQ2Y2ZiNzYz'".

    at Function (<anonymous>)
    at timeline.js.xhtml?ln=primefaces&v=8.0.1:52
    at n (timeline.js.xhtml?ln=primefaces&v=8.0.1:26)
    at timeline.js.xhtml?ln=primefaces&v=8.0.1:52
    at timeline.js.xhtml?ln=primefaces&v=8.0.1:26
    at timeline.js.xhtml?ln=primefaces&v=8.0.1:26
..

## 4) Steps to reproduce
[primefaces-test_timeline_bug_pf801.zip](https://github.com/primefaces/primefaces/files/4587932/primefaces-test_timeline_bug_pf801.zip)
.. 

## 5) Sample XHTML & ## 6) Sample bean
Sample is attached here. It is primefaces-test project and added there Timeline example from PF Showcase (basic.xhtml & BasicTimelineView.java)

Added to the web.xml:
```xml
    <context-param>
        <param-name>primefaces.CSP</param-name>
        <param-value>true</param-value>
    </context-param>
```
Changed PF version (I have PF elite licence) in maven pom:
```xml
        <dependency>
            <groupId>org.primefaces</groupId>
            <artifactId>primefaces</artifactId>
            <!--<version>8.0</version>-->
            <version>8.0.1</version>
        </dependency>
```

**Comment by melloware** [comment]
I will investigate. I use Timeline heavily with CSP but did not run into this issue so your sample must be doing something differently than I am.  I will let you know what I find out.

**Comment by melloware** [comment]
Figured it out and documented it on vis-timeline project: https://github.com/visjs/vis-timeline/issues/466

 I will get this fixed for 8.0.2
